packages/scope-manager/src/referencer/ClassVisitor.ts

```diff
@@ -229,9 +233,11 @@ class ClassVisitor extends Visitor {
     this.#referencer.close(node);
   }
 
-  protected visitProperty(
+  protected visitPropertyBase(
```

Comment:
Curiosity: why the rename?

Reply:
Because when I was adding support for accessor properties i accidentally referenced this method without realising that it was the wrong one. Took me a few minutes to figure out that it wasn't the right one.

so renaming to better show that it isn't the correct one to be calling for most cases.